src/pages/Course.vue

```diff
@@ -0,0 +1,164 @@
+      <h2>More Content to Explore</h2>
+      <div class="flex flex-wrap items-start mb5 mt4" >
+        <ButtonLink
+          v-for="course in otherCourses" :key="course.id"
+          data-cy="all-tutorials"
+          class="bg-navy white mb3 mr3"
+          text="All Tutorials"
+          :to="course.path"
+        >
+          <img
+            class="mr2"
+            :src="course.logo"
+            :alt="`${course.name} project logo`"
+            style="height: 1.5em;"
+          />
+          <span>{{course.name}} Course</span>
+        </ButtonLink>
+        <ButtonLink
+          data-cy="all-tutorials"
+          class="bg-navy white"
+          text="All Tutorials"
+          link="Tutorials"
+        />
+      </div>
+    </section>
+  </div>
+</template>
+
+<script>
+import head from '../utils/head'
+import tutorials, { correctedCases, getTutorialType } from '../utils/tutorials'
+import { getCourseNames } from '../utils/courses'
+import settings from '../utils/settings'
+import { courseList, filterTutorials } from '../utils/filters'
+import translations from '../static/translations'
+import { getAll } from '../utils/projects'
+
+import Header from '../components/Header.vue'
+import TutorialsGrid from '../components/TutorialsGrid.vue'
+import ToggleButton from '../components/ToggleButton.vue'
+import ButtonLink from '../components/buttons/ButtonLink.vue'
+import countly from '../utils/countly'
+
+export default {
+  name: 'Tutorials',
+  props: {
+    courseUrl: String
+  },
+  components: {
+    Header,
+    TutorialsGrid,
+    ToggleButton,
+    ButtonLink
+  },
+  computed: {
+    course: function () {
+      const course = this.courseUrl
+
+      // If courseUrl is not valid, redirect to 404 page
+      if (!getCourseNames().includes(course)) {
+        this.$router.replace({ name: '404' })
+
+        return null
+      }
+
+      return course
+    },
+    filteredTutorials: function () {
+      return filterTutorials(this.courseFilter, this.showCodingTutorials)
+    },
+    trackingData: function () {
+      return {
+        path: this.$route.path
+      }
+    },
+    courseName: function () {
+      return this.course.split('-').map(word => (
+        correctedCases[word] ? correctedCases[word] : (word.charAt(0).toUpperCase() + word.slice(1))
+      )).join(' ')
+    },
+    courseFilter: function () {
+      return courseList.find(course => course.key === this.course)
+    },
+    hasCodingTutorials: function () {
+      return this.courseFilter.tutorials.some(tutorialId => getTutorialType(tutorialId) === 'code' || getTutorialType(tutorialId) === 'file-upload')
+    },
+    courseDescription: function () {
+      return translations.courses[this.course].description
+    },
+    seoDescription: function () {
+      return translations.courses[this.course].seoDescription
+    },
+    otherCourses: function () {
+      return getAll().filter(course => course.id !== this.course && getCourseNames().includes(course.id))
```

Comment:
Here we are making the assumption that projects are somewhat equivalent to courses. Which is somewhat true in a way.

Although I think we could do better in making this distinction and make the actual data reflect the difference, but we can leave that for future work as a mini-refactor.